### ISSUE TITLE
Remove mypy run implicit option temporarily implemented in PR 1790 after PR 1769 gets merged

### ISSUE THREAD

**Issue by valeriupredoi** [ESMValGroup/ESMValCore#1789]
mypy command line opts put in in #1790 but after #1769 gets merged it needs to be removed :whale:

**Comment by valeriupredoi** [comment]
we are not going the pin way, instead we are setting some command args to mypy in #1790

**Comment by schlunma** [comment]
But I think we still should remove the option `implicit_optional` again after #1769.

**Comment by valeriupredoi** [comment]
indeed, good point, Manu! I renamed/redescribed and reopned, I hope I don't get rebuffed :laughing:
